lib/util/git/parent-sha-cache.ts

```diff
@@ -13,3 +14,22 @@ export function getCachedBranchParentShaResult(
 
   return null;
 }
+
+export function setCachedBranchParentShaResult(
+  branchName: string,
+  parentSha: string
+): void {
```

Comment:
```suggestion
export function setCachedBranchParentShaResult(
  branchName: string,
  branchSha: string,
  parentSha: string,
  isConflicted = false,
  isStale = false,
  isModified = false
): void {
```

I think this is the correct signature we need. I'm not sure it belongs in this file.

When a commit is made by us, the branch is not conflicted, stale, or modified.